/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

/*
 * Biology.java
 *
 * Created on 28 Jul, 2015, 2:39:52 PM
 */

/**
 *
 * @author megha
 */
public class Biology extends javax.swing.JFrame {

    /** Creates new form Biology */
    public Biology() {
        initComponents();
    }

    /** This method is called from within the constructor to
     * initialize the form.
     * WARNING: Do NOT modify this code. The content of this method is
     * always regenerated by the Form Editor.
     */
    @SuppressWarnings("unchecked")
    // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
    private void initComponents() {

        jFrame1 = new javax.swing.JFrame();
        jPanel1 = new javax.swing.JPanel();
        jLabel2 = new javax.swing.JLabel();
        jSeparator1 = new javax.swing.JSeparator();
        jLabel3 = new javax.swing.JLabel();
        jPanel2 = new javax.swing.JPanel();
        jLabel4 = new javax.swing.JLabel();
        jLabel5 = new javax.swing.JLabel();
        jPanel3 = new javax.swing.JPanel();
        jLabel6 = new javax.swing.JLabel();
        jPanel4 = new javax.swing.JPanel();
        jLabel7 = new javax.swing.JLabel();
        jLabel8 = new javax.swing.JLabel();
        jPanel5 = new javax.swing.JPanel();
        jLabel9 = new javax.swing.JLabel();
        jPanel6 = new javax.swing.JPanel();
        jLabel10 = new javax.swing.JLabel();
        jPanel8 = new javax.swing.JPanel();
        jLabel11 = new javax.swing.JLabel();
        jLabel12 = new javax.swing.JLabel();
        jLabel13 = new javax.swing.JLabel();
        jLabel14 = new javax.swing.JLabel();
        jLabel1 = new javax.swing.JLabel();
        jFrame2 = new javax.swing.JFrame();
        jPanel9 = new javax.swing.JPanel();
        jLabel17 = new javax.swing.JLabel();
        jSeparator2 = new javax.swing.JSeparator();
        jLabel19 = new javax.swing.JLabel();
        jLabel23 = new javax.swing.JLabel();
        jLabel20 = new javax.swing.JLabel();
        jLabel21 = new javax.swing.JLabel();
        jLabel22 = new javax.swing.JLabel();
        jLabel24 = new javax.swing.JLabel();
        jLabel25 = new javax.swing.JLabel();
        jLabel26 = new javax.swing.JLabel();
        jLabel27 = new javax.swing.JLabel();
        jLabel28 = new javax.swing.JLabel();
        jLabel15 = new javax.swing.JLabel();
        jFrame3 = new javax.swing.JFrame();
        jPanel10 = new javax.swing.JPanel();
        jLabel30 = new javax.swing.JLabel();
        jLabel31 = new javax.swing.JLabel();
        jSeparator3 = new javax.swing.JSeparator();
        jLabel32 = new javax.swing.JLabel();
        jLabel33 = new javax.swing.JLabel();
        jLabel34 = new javax.swing.JLabel();
        jLabel35 = new javax.swing.JLabel();
        jLabel36 = new javax.swing.JLabel();
        jLabel37 = new javax.swing.JLabel();
        jLabel29 = new javax.swing.JLabel();
        jFrame4 = new javax.swing.JFrame();
        jPanel11 = new javax.swing.JPanel();
        jLabel39 = new javax.swing.JLabel();
        jSeparator4 = new javax.swing.JSeparator();
        jLabel40 = new javax.swing.JLabel();
        jLabel41 = new javax.swing.JLabel();
        jLabel44 = new javax.swing.JLabel();
        jLabel38 = new javax.swing.JLabel();
        jFrame5 = new javax.swing.JFrame();
        jPanel12 = new javax.swing.JPanel();
        jLabel43 = new javax.swing.JLabel();
        jLabel45 = new javax.swing.JLabel();
        jLabel46 = new javax.swing.JLabel();
        jLabel47 = new javax.swing.JLabel();
        jLabel48 = new javax.swing.JLabel();
        jLabel49 = new javax.swing.JLabel();
        jLabel42 = new javax.swing.JLabel();
        jPanel7 = new javax.swing.JPanel();
        jLabel18 = new javax.swing.JLabel();
        jButton8 = new javax.swing.JButton();
        jButton9 = new javax.swing.JButton();
        jButton10 = new javax.swing.JButton();
        jButton11 = new javax.swing.JButton();
        jButton13 = new javax.swing.JButton();
        jButton12 = new javax.swing.JButton();
        jButton14 = new javax.swing.JButton();
        jButton15 = new javax.swing.JButton();
        jLabel16 = new javax.swing.JLabel();

        jPanel1.setLayout(new org.netbeans.lib.awtextra.AbsoluteLayout());

        jLabel2.setFont(new java.awt.Font("AR CENA", 0, 58));
        jLabel2.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel2.setText("Biomolecules");
        jLabel2.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0), 2));
        jPanel1.add(jLabel2, new org.netbeans.lib.awtextra.AbsoluteConstraints(430, 20, 290, 60));

        jSeparator1.setBackground(new java.awt.Color(51, 51, 51));
        jSeparator1.setForeground(new java.awt.Color(0, 0, 0));
        jSeparator1.setOrientation(javax.swing.SwingConstants.VERTICAL);
        jPanel1.add(jSeparator1, new org.netbeans.lib.awtextra.AbsoluteConstraints(570, 80, 10, 30));

        jLabel3.setBackground(new java.awt.Color(0, 0, 0));
        jLabel3.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel3.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel3.setText("All the carbon compounds which are obtained from living tissues ");
        jLabel3.setBorder(javax.swing.BorderFactory.createEtchedBorder());
        jPanel1.add(jLabel3, new org.netbeans.lib.awtextra.AbsoluteConstraints(400, 110, 370, 30));

        jPanel2.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Amino Acids", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel2.setOpaque(false);

        jLabel4.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel4.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel4.setText("<html>The amino acids are substituted methanes. <br>Amino acids are organic compounds which contain an amino group and an acidic group as substituents on the same carbon ");

        jLabel5.setIcon(new javax.swing.ImageIcon(getClass().getResource("/amino.png"))); // NOI18N

        javax.swing.GroupLayout jPanel2Layout = new javax.swing.GroupLayout(jPanel2);
        jPanel2.setLayout(jPanel2Layout);
        jPanel2Layout.setHorizontalGroup(
            jPanel2Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addGroup(jPanel2Layout.createSequentialGroup()
                .addGroup(jPanel2Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
                    .addGroup(jPanel2Layout.createSequentialGroup()
                        .addContainerGap()
                        .addComponent(jLabel4, javax.swing.GroupLayout.DEFAULT_SIZE, 268, Short.MAX_VALUE))
                    .addGroup(jPanel2Layout.createSequentialGroup()
                        .addGap(76, 76, 76)
                        .addComponent(jLabel5, javax.swing.GroupLayout.PREFERRED_SIZE, 120, javax.swing.GroupLayout.PREFERRED_SIZE)))
                .addContainerGap())
        );
        jPanel2Layout.setVerticalGroup(
            jPanel2Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addGroup(jPanel2Layout.createSequentialGroup()
                .addComponent(jLabel4, javax.swing.GroupLayout.PREFERRED_SIZE, 68, javax.swing.GroupLayout.PREFERRED_SIZE)
                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)
                .addComponent(jLabel5, javax.swing.GroupLayout.PREFERRED_SIZE, 93, Short.MAX_VALUE)
                .addContainerGap())
        );

        jPanel1.add(jPanel2, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 10, 300, 210));

        jPanel3.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Protiens", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel3.setOpaque(false);

        jLabel6.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel6.setText("<html>Polymer of amino acids<br> <br>\n<b> Primary:</b> The sequence of Amino Acids, where first amino acid is also called N-terminal amino acid. The last amino acid is called C-terminal amino acid<br> <br><b> Secondary:</b> Regularly repeating local structures gives secondary structure to protein.<br><br> <b> Tertiary:</b>The overall shape of a protein molecule; and the spatial relationship of the secondary structures to one another.");
        jLabel6.setVerticalAlignment(javax.swing.SwingConstants.TOP);

        javax.swing.GroupLayout jPanel3Layout = new javax.swing.GroupLayout(jPanel3);
        jPanel3.setLayout(jPanel3Layout);
        jPanel3Layout.setHorizontalGroup(
            jPanel3Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addGroup(jPanel3Layout.createSequentialGroup()
                .addContainerGap()
                .addComponent(jLabel6, javax.swing.GroupLayout.PREFERRED_SIZE, 253, javax.swing.GroupLayout.PREFERRED_SIZE)
                .addContainerGap(25, Short.MAX_VALUE))
        );
        jPanel3Layout.setVerticalGroup(
            jPanel3Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addGroup(jPanel3Layout.createSequentialGroup()
                .addComponent(jLabel6, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
                .addContainerGap())
        );

        jPanel1.add(jPanel3, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 220, 300, 260));

        jPanel4.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Lipids", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel4.setOpaque(false);

        jLabel7.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel7.setText("<html> Lipids are usually insoluble in water. Lipids can be simple fatty acids.\nThey can be monoglycerides, diglycerides and triglycerides.\nFatty acids could be saturated or unsaturated\n</html>");

        jLabel8.setIcon(new javax.swing.ImageIcon(getClass().getResource("/fats.png"))); // NOI18N

        javax.swing.GroupLayout jPanel4Layout = new javax.swing.GroupLayout(jPanel4);
        jPanel4.setLayout(jPanel4Layout);
        jPanel4Layout.setHorizontalGroup(
            jPanel4Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addGroup(jPanel4Layout.createSequentialGroup()
                .addGroup(jPanel4Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
                    .addGroup(jPanel4Layout.createSequentialGroup()
                        .addContainerGap()
                        .addComponent(jLabel7, javax.swing.GroupLayout.DEFAULT_SIZE, 268, Short.MAX_VALUE))
                    .addGroup(jPanel4Layout.createSequentialGroup()
                        .addGap(91, 91, 91)
                        .addComponent(jLabel8)))
                .addContainerGap())
        );
        jPanel4Layout.setVerticalGroup(
            jPanel4Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addGroup(jPanel4Layout.createSequentialGroup()
                .addComponent(jLabel7)
                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)
                .addComponent(jLabel8, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
                .addContainerGap())
        );

        jPanel1.add(jPanel4, new org.netbeans.lib.awtextra.AbsoluteConstraints(870, 10, 300, 230));

        jPanel5.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Polysaccharides", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel5.setOpaque(false);

        jLabel9.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel9.setText("<html> The long chains of sugars are called polysachharides. If a polysaccharide is made up of similar monosaccharides, it is called homopolymer.\nThe right end of a polysaccharide chain is called the reducing end and the left end is called the non-reducing end.\n</html>");

        javax.swing.GroupLayout jPanel5Layout = new javax.swing.GroupLayout(jPanel5);
        jPanel5.setLayout(jPanel5Layout);
        jPanel5Layout.setHorizontalGroup(
            jPanel5Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addGroup(jPanel5Layout.createSequentialGroup()
                .addContainerGap()
                .addComponent(jLabel9, javax.swing.GroupLayout.DEFAULT_SIZE, 268, Short.MAX_VALUE)
                .addContainerGap())
        );
        jPanel5Layout.setVerticalGroup(
            jPanel5Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addGroup(jPanel5Layout.createSequentialGroup()
                .addComponent(jLabel9)
                .addContainerGap(18, Short.MAX_VALUE))
        );

        jPanel1.add(jPanel5, new org.netbeans.lib.awtextra.AbsoluteConstraints(870, 240, 300, 150));

        jPanel6.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Nucleic Acids", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel6.setOpaque(false);

        jLabel10.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel10.setText("<html>\nA nucleic acid is composed of nucleotide. There are three chemically distinct components in a nucleotide. One of them is a heterocyclic compound, the second is a monosaccharide and the third is phosphoric acid or phosphate.\nThe heterocyclic compounds; present in nucleic acids are the nitrogenous bases, viz. adenine, guanine, uracil, cytosil and thymine. Adenine and Guanine are substituted purines, while uracil, cytosil and thymine are substituted pyrimidines.\n</html>");

        javax.swing.GroupLayout jPanel6Layout = new javax.swing.GroupLayout(jPanel6);
        jPanel6.setLayout(jPanel6Layout);
        jPanel6Layout.setHorizontalGroup(
            jPanel6Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addGroup(jPanel6Layout.createSequentialGroup()
                .addContainerGap()
                .addComponent(jLabel10, javax.swing.GroupLayout.DEFAULT_SIZE, 268, Short.MAX_VALUE)
                .addContainerGap())
        );
        jPanel6Layout.setVerticalGroup(
            jPanel6Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addGroup(jPanel6Layout.createSequentialGroup()
                .addComponent(jLabel10, javax.swing.GroupLayout.DEFAULT_SIZE, 167, Short.MAX_VALUE)
                .addContainerGap())
        );

        jPanel1.add(jPanel6, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 490, 300, 220));

        jPanel8.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Enzymes", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel8.setOpaque(false);

        jLabel11.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel11.setText("<html>An enzyme is a catalyst which is utilised in metabolic reactions. Almost all enzymes are proteins.<br> \n\n<b>Lock and Key Model:</b>Both the enzyme and the substrate possess specific complementary geometric shapes that fit exactly into one another. This model explains the specificity of enzyme. <br>\n<b>Induced Fit Model: </b>Since enzymes are rather flexible structures; the active site is continually reshaped by interactions with the substrate when the substrate interacts with the enzyme. In some cases, the substrate molecule also changes shape slightly when it enters the active site. The active site continues to change until the substrate is completely bound. The final shape and charge is determined at this point of enzyme-substrate reaction.\n");

        javax.swing.GroupLayout jPanel8Layout = new javax.swing.GroupLayout(jPanel8);
        jPanel8.setLayout(jPanel8Layout);
        jPanel8Layout.setHorizontalGroup(
            jPanel8Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addGroup(jPanel8Layout.createSequentialGroup()
                .addContainerGap()
                .addComponent(jLabel11, javax.swing.GroupLayout.DEFAULT_SIZE, 268, Short.MAX_VALUE)
                .addContainerGap())
        );
        jPanel8Layout.setVerticalGroup(
            jPanel8Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jLabel11, javax.swing.GroupLayout.DEFAULT_SIZE, 278, Short.MAX_VALUE)
        );

        jPanel1.add(jPanel8, new org.netbeans.lib.awtextra.AbsoluteConstraints(870, 400, 300, 320));

        jLabel12.setIcon(new javax.swing.ImageIcon(getClass().getResource("/1234.jpg"))); // NOI18N
        jLabel12.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0)));
        jPanel1.add(jLabel12, new org.netbeans.lib.awtextra.AbsoluteConstraints(350, 160, 250, 550));

        jLabel13.setBackground(new java.awt.Color(255, 255, 255));
        jLabel13.setIcon(new javax.swing.ImageIcon(getClass().getResource("/DNA-structure.jpg"))); // NOI18N
        jLabel13.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0)));
        jLabel13.setOpaque(true);
        jPanel1.add(jLabel13, new org.netbeans.lib.awtextra.AbsoluteConstraints(610, 160, 250, 370));

        jLabel14.setBackground(new java.awt.Color(255, 255, 255));
        jLabel14.setIcon(new javax.swing.ImageIcon(getClass().getResource("/images.png"))); // NOI18N
        jLabel14.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0)));
        jLabel14.setOpaque(true);
        jPanel1.add(jLabel14, new org.netbeans.lib.awtextra.AbsoluteConstraints(610, 540, 250, 170));

        jLabel1.setIcon(new javax.swing.ImageIcon(getClass().getResource("/green.jpg"))); // NOI18N
        jPanel1.add(jLabel1, new org.netbeans.lib.awtextra.AbsoluteConstraints(-100, -40, 1280, 770));

        javax.swing.GroupLayout jFrame1Layout = new javax.swing.GroupLayout(jFrame1.getContentPane());
        jFrame1.getContentPane().setLayout(jFrame1Layout);
        jFrame1Layout.setHorizontalGroup(
            jFrame1Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel1, javax.swing.GroupLayout.DEFAULT_SIZE, 1180, Short.MAX_VALUE)
        );
        jFrame1Layout.setVerticalGroup(
            jFrame1Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel1, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
        );

        jPanel9.setLayout(new org.netbeans.lib.awtextra.AbsoluteLayout());

        jLabel17.setFont(new java.awt.Font("AR CENA", 0, 58));
        jLabel17.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel17.setText("Genetics");
        jLabel17.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0), 2));
        jPanel9.add(jLabel17, new org.netbeans.lib.awtextra.AbsoluteConstraints(360, 10, 230, 60));

        jSeparator2.setBackground(new java.awt.Color(0, 0, 0));
        jSeparator2.setForeground(new java.awt.Color(0, 0, 0));
        jSeparator2.setOrientation(javax.swing.SwingConstants.VERTICAL);
        jPanel9.add(jSeparator2, new org.netbeans.lib.awtextra.AbsoluteConstraints(470, 70, 10, 30));

        jLabel19.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel19.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel19.setText("<html> <center>Branch of biology that studies the ways hereditary  information is passed from parent to offspring </center> </html>");
        jLabel19.setBorder(javax.swing.BorderFactory.createEtchedBorder());
        jPanel9.add(jLabel19, new org.netbeans.lib.awtextra.AbsoluteConstraints(350, 100, 250, 50));

        jLabel23.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel23.setText("<html> (i)Trisomy 21 (Down’s Syndrome) <br> (ii)Turner’s Syndrome (XO)<br> (iii)Klinefelter’s Syndrome (XXY)<br> (iv)Haemophilia<br>  (v)Sickle-cell Disease (Anemia)  ");
        jLabel23.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Disorders", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel9.add(jLabel23, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 670, 310, 120));

        jLabel20.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel20.setText("<html> <b> Law of Segregation: </b> During gamete formation, the alleles for each gene segregate from each other so that each gamete carries only one allele for each gene. <br> <br>\n<b> Law of Independent Assortment: </b> enes for different traits can segregate independently during the formation of gametes.<br> <br>\n<b>Law of Dominance: </b>Some alleles are dominant while others are recessive; an organism with at least one dominant allele will display the effect of the dominant allele.\n <br> <br>");
        jLabel20.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Mendel's Laws", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel9.add(jLabel20, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 0, 310, -1));

        jLabel21.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel21.setText("<html>\n<b> Alleles:</b> Different copies or forms a gene controlling a certain trait <br><br>\n<b>Test Cross :</b>Test to determine the unknown genotype of a parent <br><br>\n<b>Genotype:</b>The genetic makeup ( Eg: TT,Tt,tt)<br><br>\n<b>Phenotype:</b> The physical trait(Eg: Tall or dwarf)<br>\n");
        jLabel21.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel21.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Terminology", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel9.add(jLabel21, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 250, 310, 180));

        jLabel22.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel22.setText("<html>\nChange in genetic make-up<br><br>\n\n<b> Point Gene Mutation:</b> It only changes single codon triplet <br>\n<b> Frameshift Gene Mutation:</b>  All codons after change are altered as well<br><br>\n<b> Chromosomal Mutations:</b> <br>\n(i)Translocation<br>\n(ii)Inversion<br>\n(iii)Addition<br>\n(iv)Deletion<br>\n(v)Nondisjunction (polyploidy)");
        jLabel22.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel22.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Mutations", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel9.add(jLabel22, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 430, 310, 240));

        jLabel24.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel24.setText("<html>\nThe process by which DNA makes a copy of itself during cell division. <br><br>\n<b>Initiation:</b>\nBegins at a location on the double helix known as “oriC”. Enzymes helicases unwind the double helix  . The “topoisomerase” proteins surround the unzipping strands and relax the twisting that might damage the unwinding DNA. <br><br>\n<b>Elongation:</b>The enzyme DNA polymerase controls elongation, which can occur only in the leading direction. The lagging strand unwinds in small sections that DNA polymerase replicates in the leading direction.<br><br>\n<b>Termination:</b>The last portion of the lagging strand is the telomere section, containing a repeating non-coding sequence of bases. Enzymes snip off a telomere at the end of each replication.<br>");
        jLabel24.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel24.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "DNA Replication", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel9.add(jLabel24, new org.netbeans.lib.awtextra.AbsoluteConstraints(650, 0, 360, 310));

        jLabel25.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel25.setText("<html>\nGenes are responsible for making proteins in an organism<br>\nEach gene only can make one protein (ONE GENE – ONE POLYPEPTIDE)<br>\nThe DNA is arranged in a specific order to code for a specific protein<br>\nNOT ALL areas code for something (Exons vs. Introns)<br><br>\n3 Types of RNA<br>\n1.    mRNA (messenger RNA)<br>\n2.    rRNA (ribosomal RNA)<br>\n3.    tRNA (transfer RNA)\n\n\n</html>\n");
        jLabel25.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel25.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Gene Expression", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel9.add(jLabel25, new org.netbeans.lib.awtextra.AbsoluteConstraints(650, 320, 360, 210));

        jLabel26.setBackground(new java.awt.Color(255, 255, 255));
        jLabel26.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel26.setIcon(new javax.swing.ImageIcon(getClass().getResource("/replicate.jpg"))); // NOI18N
        jLabel26.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0)));
        jLabel26.setOpaque(true);
        jPanel9.add(jLabel26, new org.netbeans.lib.awtextra.AbsoluteConstraints(350, 180, 270, 430));

        jLabel27.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel27.setText("<html>\nThey are genetic regulatory system found in bacteria and their viruses in which genes coding for functionally related proteins are clustered along the DNA. This feature allows protein synthesis to be controlled coordinately in response to the needs of the cell.\n<br><br>\nThe main conponents of an operon are-<br>\n(i) <b>Promotor sequence- </b>Where RNA polymerase binds. <br>\n(ii)<b>Operator sequence-</b> Where a repressor binds and inhibits RNA polymerase from binding. <br>\n(iii)<b>Structural sequence- </b> The sequence that codes for the protein.\n</html>");
        jLabel27.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel27.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Operons", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel9.add(jLabel27, new org.netbeans.lib.awtextra.AbsoluteConstraints(650, 550, 360, 240));

        jLabel28.setBackground(new java.awt.Color(255, 255, 255));
        jLabel28.setHorizontalAlignment(javax.swing.SwingConstants.RIGHT);
        jLabel28.setIcon(new javax.swing.ImageIcon(getClass().getResource("/chromo.jpg"))); // NOI18N
        jLabel28.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0)));
        jLabel28.setOpaque(true);
        jPanel9.add(jLabel28, new org.netbeans.lib.awtextra.AbsoluteConstraints(350, 620, 270, 170));

        jLabel15.setIcon(new javax.swing.ImageIcon(getClass().getResource("/green.jpg"))); // NOI18N
        jPanel9.add(jLabel15, new org.netbeans.lib.awtextra.AbsoluteConstraints(-10, -20, 1030, 820));

        javax.swing.GroupLayout jFrame2Layout = new javax.swing.GroupLayout(jFrame2.getContentPane());
        jFrame2.getContentPane().setLayout(jFrame2Layout);
        jFrame2Layout.setHorizontalGroup(
            jFrame2Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel9, javax.swing.GroupLayout.DEFAULT_SIZE, 1020, Short.MAX_VALUE)
        );
        jFrame2Layout.setVerticalGroup(
            jFrame2Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel9, javax.swing.GroupLayout.DEFAULT_SIZE, 800, Short.MAX_VALUE)
        );

        jPanel10.setLayout(new org.netbeans.lib.awtextra.AbsoluteLayout());

        jLabel30.setFont(new java.awt.Font("AR CENA", 0, 58));
        jLabel30.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel30.setText("Evolution");
        jLabel30.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0), 2));
        jPanel10.add(jLabel30, new org.netbeans.lib.awtextra.AbsoluteConstraints(420, 10, 260, 70));

        jLabel31.setBackground(new java.awt.Color(0, 0, 0));
        jLabel31.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel31.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel31.setText("<html><center>Life developing from earlier forms during the history of the earth</center></html>");
        jLabel31.setBorder(javax.swing.BorderFactory.createEtchedBorder());
        jPanel10.add(jLabel31, new org.netbeans.lib.awtextra.AbsoluteConstraints(360, 90, 370, 30));

        jSeparator3.setBackground(new java.awt.Color(0, 0, 0));
        jSeparator3.setForeground(new java.awt.Color(0, 0, 0));
        jSeparator3.setOrientation(javax.swing.SwingConstants.VERTICAL);
        jPanel10.add(jSeparator3, new org.netbeans.lib.awtextra.AbsoluteConstraints(540, 80, 10, 10));

        jLabel32.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel32.setText("<html>\n<b>Biogenesis:</b> The idea that life can only arise from pre-existing life<br>\n<b>Heterotroph hypothesis:</b> No atmosphere on Earth.  Random chemical reactions gave rise to anaerobes which increased carbon dioxide in the atmosphere, this allowed for photosynthetic organisms to develop which then added oxygen<br>\n \n\n  <b>   Lamarck’s Theory:</b> The more an organ is used, the bigger and more developed it becomes; the less an organ is used, the small and less developed it becomes. There exists ability to pass on traits obtained during lifetime<br>\n<b>    Darwin’s Theory:</b><br>\n\n                                                             i.      Overproduction<br>\n                                                          ii.      Competition<br>\n                                                       iii.      Variation<br>\n                                                        iv.      Adaptation of any trait that improves an organisms chance for survival and reproduction<br>\n                                                           v.      Natural Selection<br>\n                                                        vi.      Speciation<br>\n</html>");
        jLabel32.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel32.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Theories", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel10.add(jLabel32, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 0, 320, 350));

        jLabel33.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel33.setText("<html>\n<b>Types of Adaptations:</b><br>\n(i)Structural-Involving the body of the organism<br>\n(ii)Physiological-Involving the metabolism of the organism<br><br>\n<b> Reason for Adaptation:</b><br>\n1.    Increasing ability to obtain food<br>\n2.    Increasing chance of reproduction<br>\n3.    Increasing ability to avoid predators<br>\na.     Camouflage<br>\nb.     Warning Coloration and Mimicry<br><br>\n     \n<b> Types of Natural Selection:</b><br>\n\n(i) Direction<br>\n(ii) Stabilizing<br>\n(iii)Disruptive\n</html>");
        jLabel33.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Adaptation and Natural Selection", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel10.add(jLabel33, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 360, 320, 290));

        jLabel34.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel34.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel34.setText("<html><b>Speciation:</b> The formation of a new , when species lose the ability to interbreed<br><br>\n     <b> Isolation:</b>Anything that prevents two groups within a species from interbreeding\n will lead to enough of a difference that reproduction between groups will no longer occur\nUsually two stages- <br>\na.     Geographic Isolation<br>\nb.     Reproductive Isolation<br><br>\n\n\n<b>Adaptive Radiation:</b>.As organisms radiate (spread out) they must adapt to new environments; hence they will develop differences which may cause speciation.<br><br>\n\n \n<b>  Convergent Evolution:</b>Non related species may evolve to appear very similar to one another even though they are not related.<br><br>\n<b>Co-evolution:</b>  Non related species may effect one another’s evolution as they evolve in response to one another.Eg:Flowers and bees.\n\n</html>\n");
        jLabel34.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel34.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Terminology", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel10.add(jLabel34, new org.netbeans.lib.awtextra.AbsoluteConstraints(750, 0, 330, 370));

        jLabel35.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel35.setText("<html>\n\n                                                          <b>     Homologous structures:</b>-Similar structure, different function<br><br>\n<b>     Analogous structures:</b>     -Similar function, different structure<br><br>\n \n \n<b>     Vestigial structures: </b>Remnants of structures that were functional in ancestral forms<br><br>\n\n<b>     Embryological Similarities:</b> Similarities in fetal development<br><br>\n\n<b>    Molecular Similarities:</b>      DNA as evidence. More similar sequences indicate relationships\n\n\n</html>");
        jLabel35.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel35.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Evidences Of Evolution", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel10.add(jLabel35, new org.netbeans.lib.awtextra.AbsoluteConstraints(750, 380, 330, 260));

        jLabel36.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel36.setIcon(new javax.swing.ImageIcon(getClass().getResource("/embryo.jpg"))); // NOI18N
        jLabel36.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0)));
        jPanel10.add(jLabel36, new org.netbeans.lib.awtextra.AbsoluteConstraints(380, 130, 320, 270));

        jLabel37.setIcon(new javax.swing.ImageIcon(getClass().getResource("/df.jpg"))); // NOI18N
        jLabel37.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0)));
        jPanel10.add(jLabel37, new org.netbeans.lib.awtextra.AbsoluteConstraints(380, 410, 320, 240));

        jLabel29.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel29.setIcon(new javax.swing.ImageIcon(getClass().getResource("/green.jpg"))); // NOI18N
        jPanel10.add(jLabel29, new org.netbeans.lib.awtextra.AbsoluteConstraints(0, 0, 1080, 660));

        javax.swing.GroupLayout jFrame3Layout = new javax.swing.GroupLayout(jFrame3.getContentPane());
        jFrame3.getContentPane().setLayout(jFrame3Layout);
        jFrame3Layout.setHorizontalGroup(
            jFrame3Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel10, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
        );
        jFrame3Layout.setVerticalGroup(
            jFrame3Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel10, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
        );

        jPanel11.setLayout(new org.netbeans.lib.awtextra.AbsoluteLayout());

        jLabel39.setFont(new java.awt.Font("AR CENA", 0, 58));
        jLabel39.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel39.setText("Cell-Basics");
        jLabel39.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0), 2));
        jPanel11.add(jLabel39, new org.netbeans.lib.awtextra.AbsoluteConstraints(420, 10, 240, 80));

        jSeparator4.setBackground(new java.awt.Color(0, 0, 0));
        jSeparator4.setForeground(new java.awt.Color(0, 0, 0));
        jSeparator4.setOrientation(javax.swing.SwingConstants.VERTICAL);
        jPanel11.add(jSeparator4, new org.netbeans.lib.awtextra.AbsoluteConstraints(540, 90, 20, 30));

        jLabel40.setBackground(new java.awt.Color(0, 0, 0));
        jLabel40.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel40.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel40.setText("The basic building blocks of all living things");
        jLabel40.setBorder(javax.swing.BorderFactory.createEtchedBorder());
        jPanel11.add(jLabel40, new org.netbeans.lib.awtextra.AbsoluteConstraints(420, 120, 250, 30));

        jLabel41.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel41.setText("<html>\n<b>Cytoplasm:</b>Within cells, the cytoplasm is made up of a jelly-like fluid (called the cytosol) and other structures that surround the nucleus.<br><br>\n<b>Cytoskeleton:</b>The cytoskeleton is a network of long fibers that make up the cell’s structural framework. The cytoskeleton has several critical functions, including determining cell shape, participating in cell division, and allowing cells to move. It also provides a track-like system that directs the movement of organelles and other substances within cells.<br><br>\n<b>Endoplasmic Reticulum(ER):</b>This organelle helps process molecules created by the cell. The ER also transports these molecules to their specific destinations either inside or outside the cell.<br><br>\n<b>Golgi apparatus:</b>The Golgi apparatus packages molecules processed by the endoplasmic reticulum to be transported out of the cell.<br><br>\n<b>Lysosomes and peroxisomes :</b>These organelles are the recycling center of the cell. They digest foreign bacteria that invade the cell, rid the cell of toxic substances, and recycle worn-out cell components.<br><br>\n<b>Mitochondria:</b>Mitochondria are complex organelles that convert energy from food into a form that the cell can use. They have their own genetic material, separate from the DNA in the nucleus, and can make copies of themselves.<br><br>\n<b>Nucleus:</b>The nucleus serves as the cell’s command center, sending directions to the cell to grow, mature, divide, or die. It also houses DNA,the hereditary material. The nucleus is surrounded by a membrane called the nuclear envelope, which protects the DNA and separates the nucleus from the rest of the cell.<br><br>\n<b>Plasma membrane:</b>The plasma membrane is the outer lining of the cell. It separates the cell from its environment and allows materials to enter and leave the cell.<br><br>\n<b>Ribosomes :</b>Ribosomes are organelles that process the cell’s genetic instructions to create proteins. These organelles can float freely in the cytoplasm or be connected to the ER<br><br>\n</html>");
        jLabel41.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel41.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Parts", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel11.add(jLabel41, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 0, 350, 720));

        jLabel44.setBackground(new java.awt.Color(255, 255, 255));
        jLabel44.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel44.setIcon(new javax.swing.ImageIcon(getClass().getResource("/cell.jpg"))); // NOI18N
        jLabel44.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0), 2));
        jLabel44.setOpaque(true);
        jPanel11.add(jLabel44, new org.netbeans.lib.awtextra.AbsoluteConstraints(370, 170, 610, 550));

        jLabel38.setIcon(new javax.swing.ImageIcon(getClass().getResource("/green.jpg"))); // NOI18N
        jPanel11.add(jLabel38, new org.netbeans.lib.awtextra.AbsoluteConstraints(0, 0, 990, 740));

        javax.swing.GroupLayout jFrame4Layout = new javax.swing.GroupLayout(jFrame4.getContentPane());
        jFrame4.getContentPane().setLayout(jFrame4Layout);
        jFrame4Layout.setHorizontalGroup(
            jFrame4Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel11, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
        );
        jFrame4Layout.setVerticalGroup(
            jFrame4Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel11, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
        );

        jPanel12.setLayout(new org.netbeans.lib.awtextra.AbsoluteLayout());

        jLabel43.setFont(new java.awt.Font("AR CENA", 0, 58));
        jLabel43.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel43.setText("Biotechnology");
        jLabel43.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0), 2));
        jPanel12.add(jLabel43, new org.netbeans.lib.awtextra.AbsoluteConstraints(340, 0, 280, 60));

        jLabel45.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel45.setText("<html>\n<ol><li>Biotechnology refers to technology used to manipulate DNA<br>\n<li>The procedures are often referred to as genetic engineering<br>\n<li>Genes from one kind of organism can be transcribed and translated when put into another kind of organism<br>\n<li>For example, human and other genes are routinely put into bacteria in order to synthesize products for medical treatment and commercial use<br>\n<li>Human insulin, human growth hormone, and vaccines are produced by bacteria<br>\n<li>Recombinant DNA refers to DNA from two different source<br>\n<li>Individuals that receive genes from other species are transgenic<br>\n</ul>\n</html>");
        jLabel45.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel45.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Description", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel12.add(jLabel45, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 0, 320, 310));

        jLabel46.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel46.setText("<html>\nVectors are used to transfer genes into a host cell.\nPlasmids & viruses are the most commonly used vectors.\nA vector must be capable of self-replicating inside a cell.\nViruses are the vectors of choice for animal cells.\nMarker genes can be used to determine if the gene has been taken up. <br>\n<ul><li><b> Plasmids:</b>Small rings of DNA in bacterial cells,used to transfer genes to other organisms.<br>\n<li><b>Viruses:</b>Can accept larger amounts of DNA than plasmids\nOnce the virus enters the host cell, it also reproduces the foreign gene it carries.\n</ul>\n<b> Restriction Enzymes:</b> Restriction enzymes were discovered in bacteria, and are used to cut DNA at a specific base sequence. Eg: EcoRI always cuts DNA at GAATTC.<br><br>\n</html>");
        jLabel46.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel46.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Vectors", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel12.add(jLabel46, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 310, 320, 330));

        jLabel47.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel47.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel47.setText("<html><center>Used to make many copies of small pieces of DNA<br>\nProcedure requires primers, DNA polymerase, and nucleotides<br>\nPrimers are short chains of about 20 nucleotides that are complimentary to a region in the DNA to be amplified<br>\nDNA polymerase cannot continue the process unless it has already been started by primers<br>\nNucleotides are needed because DNA is composed of nucleotide \"building blocks\"<br>\nThe DNA is heated to approximately 95<sup>o</sup> C to separate the two strands of the double helix\nAfter the strands are separated, the DNA is cooled to about 50<sup>o </sup>C, and the primers attach<br>\nThe temperature is raised to approximately 70<sup>o</sup>C so the polymerase will attach to & copy the strand<br>\nThe DNA replication process repeats itself as the solution is then heated and cooled at regular intervals\n</center></html>");
        jLabel47.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel47.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Polymerase Chain Reaction", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel12.add(jLabel47, new org.netbeans.lib.awtextra.AbsoluteConstraints(630, 0, 310, 330));

        jLabel48.setFont(new java.awt.Font("Tahoma", 0, 12));
        jLabel48.setText("<html>\n\n<b>Sticky Ends & Recombinant DNA:</b><br>\n<ul>\n\n<li>If the vector and the gene to be cloned are both cut with the same restriction enzyme, they will both have complimentary sticky ends<br>\n\n<li>After cutting, the 2 DNA samples are mixed<br>\n\n<li>Fragments with complementary sticky ends join together forming recombinant DNA <br>\n<li>Enzyme DNA ligase seals the fragments together<br>\n\n<li>Bacteria such as E. coli are capable of taking up DNA from their environment by transformation.<br>\n\n<li>CaCl2 and a procedure called heat shock are used to make E. coli cells more permeable so that they take up the modified plasmids more readily.\n</ul>\n</html>");
        jLabel48.setVerticalAlignment(javax.swing.SwingConstants.TOP);
        jLabel48.setBorder(javax.swing.BorderFactory.createTitledBorder(null, "Tools and Methods", javax.swing.border.TitledBorder.DEFAULT_JUSTIFICATION, javax.swing.border.TitledBorder.DEFAULT_POSITION, new java.awt.Font("AR CENA", 0, 24))); // NOI18N
        jPanel12.add(jLabel48, new org.netbeans.lib.awtextra.AbsoluteConstraints(630, 330, 310, 310));

        jLabel49.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel49.setIcon(new javax.swing.ImageIcon(getClass().getResource("/bio.jpg"))); // NOI18N
        jLabel49.setBorder(javax.swing.BorderFactory.createLineBorder(new java.awt.Color(0, 0, 0), 2));
        jPanel12.add(jLabel49, new org.netbeans.lib.awtextra.AbsoluteConstraints(340, 70, 280, 580));

        jLabel42.setIcon(new javax.swing.ImageIcon(getClass().getResource("/green.jpg"))); // NOI18N
        jPanel12.add(jLabel42, new org.netbeans.lib.awtextra.AbsoluteConstraints(-10, 0, 960, 650));

        javax.swing.GroupLayout jFrame5Layout = new javax.swing.GroupLayout(jFrame5.getContentPane());
        jFrame5.getContentPane().setLayout(jFrame5Layout);
        jFrame5Layout.setHorizontalGroup(
            jFrame5Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel12, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
        );
        jFrame5Layout.setVerticalGroup(
            jFrame5Layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel12, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE)
        );

        setDefaultCloseOperation(javax.swing.WindowConstants.EXIT_ON_CLOSE);

        jPanel7.setLayout(new org.netbeans.lib.awtextra.AbsoluteLayout());

        jLabel18.setFont(new java.awt.Font("Matura MT Script Capitals", 0, 58));
        jLabel18.setForeground(new java.awt.Color(255, 255, 255));
        jLabel18.setText("Biology");
        jPanel7.add(jLabel18, new org.netbeans.lib.awtextra.AbsoluteConstraints(320, 10, 270, 70));

        jButton8.setFont(new java.awt.Font("AR CENA", 0, 24));
        jButton8.setForeground(new java.awt.Color(255, 255, 255));
        jButton8.setText("Evolution");
        jButton8.setBorder(javax.swing.BorderFactory.createEtchedBorder());
        jButton8.setContentAreaFilled(false);
        jButton8.addActionListener(new java.awt.event.ActionListener() {
            public void actionPerformed(java.awt.event.ActionEvent evt) {
                jButton8ActionPerformed(evt);
            }
        });
        jPanel7.add(jButton8, new org.netbeans.lib.awtextra.AbsoluteConstraints(20, 170, 260, 40));

        jButton9.setFont(new java.awt.Font("AR CENA", 0, 24));
        jButton9.setForeground(new java.awt.Color(255, 255, 255));
        jButton9.setText("Biomolecules");
        jButton9.setBorder(javax.swing.BorderFactory.createEtchedBorder());
        jButton9.setContentAreaFilled(false);
        jButton9.addActionListener(new java.awt.event.ActionListener() {
            public void actionPerformed(java.awt.event.ActionEvent evt) {
                jButton9ActionPerformed(evt);
            }
        });
        jPanel7.add(jButton9, new org.netbeans.lib.awtextra.AbsoluteConstraints(20, 220, 260, 40));

        jButton10.setFont(new java.awt.Font("AR CENA", 0, 24));
        jButton10.setForeground(new java.awt.Color(255, 255, 255));
        jButton10.setText("Cell: Basics");
        jButton10.setBorder(javax.swing.BorderFactory.createEtchedBorder());
        jButton10.setContentAreaFilled(false);
        jButton10.addActionListener(new java.awt.event.ActionListener() {
            public void actionPerformed(java.awt.event.ActionEvent evt) {
                jButton10ActionPerformed(evt);
            }
        });
        jPanel7.add(jButton10, new org.netbeans.lib.awtextra.AbsoluteConstraints(20, 270, 260, 40));

        jButton11.setFont(new java.awt.Font("AR CENA", 0, 24));
        jButton11.setForeground(new java.awt.Color(255, 255, 255));
        jButton11.setText("Biotechnology");
        jButton11.setBorder(javax.swing.BorderFactory.createEtchedBorder());
        jButton11.setContentAreaFilled(false);
        jButton11.addActionListener(new java.awt.event.ActionListener() {
            public void actionPerformed(java.awt.event.ActionEvent evt) {
                jButton11ActionPerformed(evt);
            }
        });
        jPanel7.add(jButton11, new org.netbeans.lib.awtextra.AbsoluteConstraints(20, 320, 260, 40));

        jButton13.setFont(new java.awt.Font("AR CENA", 0, 24));
        jButton13.setForeground(new java.awt.Color(255, 255, 255));
        jButton13.setText("Genetics");
        jButton13.setBorder(javax.swing.BorderFactory.createEtchedBorder());
        jButton13.setContentAreaFilled(false);
        jButton13.addActionListener(new java.awt.event.ActionListener() {
            public void actionPerformed(java.awt.event.ActionEvent evt) {
                jButton13ActionPerformed(evt);
            }
        });
        jPanel7.add(jButton13, new org.netbeans.lib.awtextra.AbsoluteConstraints(20, 120, 260, 40));

        jButton12.setFont(new java.awt.Font("Tahoma", 1, 14));
        jButton12.setText("Ask a Question");
        jButton12.setBorderPainted(false);
        jButton12.setContentAreaFilled(false);
        jButton12.addActionListener(new java.awt.event.ActionListener() {
            public void actionPerformed(java.awt.event.ActionEvent evt) {
                jButton12ActionPerformed(evt);
            }
        });
        jPanel7.add(jButton12, new org.netbeans.lib.awtextra.AbsoluteConstraints(10, 480, 210, 60));

        jButton14.setFont(new java.awt.Font("Tahoma", 1, 14));
        jButton14.setText("Take a Test");
        jButton14.setBorderPainted(false);
        jButton14.setContentAreaFilled(false);
        jPanel7.add(jButton14, new org.netbeans.lib.awtextra.AbsoluteConstraints(350, 480, 190, 60));

        jButton15.setFont(new java.awt.Font("Tahoma", 1, 14));
        jButton15.setText("Back to Subjects");
        jButton15.setBorderPainted(false);
        jButton15.setContentAreaFilled(false);
        jPanel7.add(jButton15, new org.netbeans.lib.awtextra.AbsoluteConstraints(660, 490, -1, 50));

        jLabel16.setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
        jLabel16.setIcon(new javax.swing.ImageIcon(getClass().getResource("/6240_1280x800.jpg"))); // NOI18N
        jPanel7.add(jLabel16, new org.netbeans.lib.awtextra.AbsoluteConstraints(0, 0, 880, 540));

        javax.swing.GroupLayout layout = new javax.swing.GroupLayout(getContentPane());
        getContentPane().setLayout(layout);
        layout.setHorizontalGroup(
            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel7, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
        );
        layout.setVerticalGroup(
            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
            .addComponent(jPanel7, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
        );

        pack();
    }// </editor-fold>//GEN-END:initComponents

    private void jButton13ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jButton13ActionPerformed
        // TODO add your handling code here:
        jFrame2.setVisible(true);


}//GEN-LAST:event_jButton13ActionPerformed

    private void jButton12ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jButton12ActionPerformed
        // TODO add your handling code here:
}//GEN-LAST:event_jButton12ActionPerformed

    private void jButton9ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jButton9ActionPerformed
        // TODO add your handling code here:
        jFrame1.setVisible(true);
        
    }//GEN-LAST:event_jButton9ActionPerformed

    private void jButton8ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jButton8ActionPerformed
        // TODO add your handling code here:
        jFrame3.setVisible(true);
    }//GEN-LAST:event_jButton8ActionPerformed

    private void jButton10ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jButton10ActionPerformed
        // TODO add your handling code here:
        jFrame4.setVisible(true);
    }//GEN-LAST:event_jButton10ActionPerformed

    private void jButton11ActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jButton11ActionPerformed
        // TODO add your handling code here:
        jFrame5.setVisible(true);
    }//GEN-LAST:event_jButton11ActionPerformed

    /**
    * @param args the command line arguments
    */
    public static void main(String args[]) {
        java.awt.EventQueue.invokeLater(new Runnable() {
            public void run() {
                new Biology().setVisible(true);
            }
        });
    }

    // Variables declaration - do not modify//GEN-BEGIN:variables
    private javax.swing.JButton jButton10;
    private javax.swing.JButton jButton11;
    private javax.swing.JButton jButton12;
    private javax.swing.JButton jButton13;
    private javax.swing.JButton jButton14;
    private javax.swing.JButton jButton15;
    private javax.swing.JButton jButton8;
    private javax.swing.JButton jButton9;
    private javax.swing.JFrame jFrame1;
    private javax.swing.JFrame jFrame2;
    private javax.swing.JFrame jFrame3;
    private javax.swing.JFrame jFrame4;
    private javax.swing.JFrame jFrame5;
    private javax.swing.JLabel jLabel1;
    private javax.swing.JLabel jLabel10;
    private javax.swing.JLabel jLabel11;
    private javax.swing.JLabel jLabel12;
    private javax.swing.JLabel jLabel13;
    private javax.swing.JLabel jLabel14;
    private javax.swing.JLabel jLabel15;
    private javax.swing.JLabel jLabel16;
    private javax.swing.JLabel jLabel17;
    private javax.swing.JLabel jLabel18;
    private javax.swing.JLabel jLabel19;
    private javax.swing.JLabel jLabel2;
    private javax.swing.JLabel jLabel20;
    private javax.swing.JLabel jLabel21;
    private javax.swing.JLabel jLabel22;
    private javax.swing.JLabel jLabel23;
    private javax.swing.JLabel jLabel24;
    private javax.swing.JLabel jLabel25;
    private javax.swing.JLabel jLabel26;
    private javax.swing.JLabel jLabel27;
    private javax.swing.JLabel jLabel28;
    private javax.swing.JLabel jLabel29;
    private javax.swing.JLabel jLabel3;
    private javax.swing.JLabel jLabel30;
    private javax.swing.JLabel jLabel31;
    private javax.swing.JLabel jLabel32;
    private javax.swing.JLabel jLabel33;
    private javax.swing.JLabel jLabel34;
    private javax.swing.JLabel jLabel35;
    private javax.swing.JLabel jLabel36;
    private javax.swing.JLabel jLabel37;
    private javax.swing.JLabel jLabel38;
    private javax.swing.JLabel jLabel39;
    private javax.swing.JLabel jLabel4;
    private javax.swing.JLabel jLabel40;
    private javax.swing.JLabel jLabel41;
    private javax.swing.JLabel jLabel42;
    private javax.swing.JLabel jLabel43;
    private javax.swing.JLabel jLabel44;
    private javax.swing.JLabel jLabel45;
    private javax.swing.JLabel jLabel46;
    private javax.swing.JLabel jLabel47;
    private javax.swing.JLabel jLabel48;
    private javax.swing.JLabel jLabel49;
    private javax.swing.JLabel jLabel5;
    private javax.swing.JLabel jLabel6;
    private javax.swing.JLabel jLabel7;
    private javax.swing.JLabel jLabel8;
    private javax.swing.JLabel jLabel9;
    private javax.swing.JPanel jPanel1;
    private javax.swing.JPanel jPanel10;
    private javax.swing.JPanel jPanel11;
    private javax.swing.JPanel jPanel12;
    private javax.swing.JPanel jPanel2;
    private javax.swing.JPanel jPanel3;
    private javax.swing.JPanel jPanel4;
    private javax.swing.JPanel jPanel5;
    private javax.swing.JPanel jPanel6;
    private javax.swing.JPanel jPanel7;
    private javax.swing.JPanel jPanel8;
    private javax.swing.JPanel jPanel9;
    private javax.swing.JSeparator jSeparator1;
    private javax.swing.JSeparator jSeparator2;
    private javax.swing.JSeparator jSeparator3;
    private javax.swing.JSeparator jSeparator4;
    // End of variables declaration//GEN-END:variables

}
